lighthouse-core/gather/gather-runner.js

```diff
@@ -55,17 +58,27 @@ const NetworkRecorder = require('../lib/network-recorder.js');
  */
 class GatherRunner {
   /**
-   * Loads about:blank and waits there briefly. Since a Page.reload command does
-   * not let a service worker take over, we navigate away and then come back to
-   * reload. We do not `waitForLoad` on about:blank since a page load event is
-   * never fired on it.
+   * Loads a blank page and waits there briefly. Since a Page.reload command does
+   * not let a service worker take over, we navigate away and then come back to reload.
    * @param {!Driver} driver
    * @param {url=} url
    * @param {number=} duration
    * @return {!Promise}
    */
-  static loadBlank(driver, url = 'about:blank', duration = 300) {
-    return driver.gotoURL(url).then(_ => new Promise(resolve => setTimeout(resolve, duration)));
+  static loadBlank(driver, url) {
+    // The real about:blank doesn't fire onload and is full of mysteries (https://goo.gl/mdQkYr)
+    // To improve speed and avoid anomalies (https://goo.gl/Aho2R9), we use a basic data uri page
+    url = url || `data:text/html,${logoPageSource}`;
+    const blankPageUrl = `data:text/html,${blankPageSource}`;
+
+    // Only navigating to a single data-uri doesn't reliably trigger onload. (Why? Beats me.)
+    // Two data uris work, however the two need to be sufficiently different (Why? Beats me.)
+    // If they are too similar, Chrome considers the latter to be as superficial as a pushState
+    // Lastly, it's possible for two navigations to be racy, so we await onload inbetween.
```

Comment:
Maybe throw a note in here that it's still not that long since onload happens real fast for these suckers.

Also, if we eliminate the double `loadBlank` call at the beginning we would've been fine too, right?

Reply:
> Also, if we eliminate the double loadBlank call at the beginning we would've been fine too, right?

Hmmm interesting. I gotta look into why we added it: https://github.com/GoogleChrome/lighthouse/pull/850

Reply:
well I meant introducing something more like "if this is the first pass, don't bother trying to navigate to blank since we know we're already on it" not necessarily nuking the first one :)

Reply:
is https://github.com/GoogleChrome/lighthouse/pull/4310/commits/df175bc06409fe29b7eb24a76e37d89a816d3703 what you were thinking?

Reply:
ah yes, it is! though I believe in LR they had to switch the order of passes so we can't necessarily rely on `defaultPass` being first :/ an index approach seems annoying though. 

I'm cool to revisit later if you think that's too much trouble since we've already reduced the time considerably.

Reply:
yeah sg. i'll do a followup PR to use the index.

Reply:
included passIndex in this PR. 2 line change. :)